src/components/dialogs/exit-incomplete-payment.vue

```diff
@@ -1,21 +1,26 @@
 <template>
   <v-dialog v-model="showModal" max-width="45rem">
     <v-card>
-      <v-card-title class="d-flex justify-space-between mt-n3">
```

Comment:
Since I used this component (and others) as a template for the Affiliation Error dialog, I fixed up the layout in this component for consistency.